addons/base_geolocalize/models/res_partner.py

```diff
@@ -31,4 +32,11 @@ def geo_localize(self):
                     'partner_longitude': result[1],
                     'date_localization': fields.Date.context_today(partner)
                 })
+            else:
+                partners_not_geo_localized |= partner
+        if partners_not_geo_localized:
+            self.env['bus.bus'].sendone(
```

Comment:
Linting: just to ease reading, could be multi-lined (+ use named parameters for string formatting, helps reading it)
```
    self.env['bus.bus'].sendone(
        (self.env.cr.dbname, 'res.partner', self.env.user.partner_id.id),
        {'type': 'simple_notification',
         'title': _("Warning"),
         'message': _('No address found for %(partner_names)s.',
                      partner_names=', '.join(partners_not_geo_localized.mapped('name'))
                     )
            }
```